README.md

```diff
@@ -185,6 +185,29 @@ async def purge(ctx: commands.Context, tz_info: str, military_time: bool):
 ```
 </details>
 
+<details><summary><b>Asset handler</b></summary>
+In order to prevent the transcripts from being broken either when a channel is deleted or by the newly introduced 
+restrictions to media links in discord, chat-exporter now supports an asset handler. `chat_exporter.construct.
+asset_handler.AssetHandler` servers as a template for you to implement your own asset handler. As example we provide 
+to basic versions of an asset handler, one that stores the assets locally and one that uploads them to a discord. Of 
+course the second one is also broken, but it should give a good idea on how to implement such an `AssetHandler`.
+If you dont specify an asset handler, chat-exporter will use the normal (proxy) urls for the assets.
+The important part of your implementation is, that you have to overwrite the url and proxy_url attribute of the 
+Attachment in your implementation of `AssetHandler`. The url attribute should be the url where the asset is available.
```

Comment:
Can you split this up please? Block paragraphs aren't good for anyones mental health 👀